CorsixTH/Lua/humanoid_actions/walk.lua

```diff
@@ -112,7 +112,7 @@ action_walk_interrupt = permanent"action_walk_interrupt"( function(action, human
 end)
 
 local flag_list_bottom = 2048
-local flag_early_list = 1024
+--  flag_early_list = 1024
```

Comment:
Documentation purposes, mostly. I don't know another place where those bit numbers and names/explanation are related with each other.
Note that I also don't know if the complicated way of building the screen is still used/useful; using OpenGL blitting opens a lot of other ways of doing things, like the Z-buffer.